sunpy/net/attr.py

```diff
@@ -60,6 +137,74 @@ def collides(self, other):
+        We sanitize the name you provide by removing all special characters and making it all lower case.
+        If it still invalid we will append to the start of the name to make it a valid attribute name.
+        This is an ugly workaround, so please try to pick a valid attribute name.
+
+        **It is recommended that clients register values for all attrs used by their `_can_handle_query` method.**
+
+        Paramaters
+        ----------
+
+        adcit : `dict`
+            A dictionary that has keys of `~sunpy.net.attr.Attr`.
+            Each key should have a list of tuples as it value.
+            Each tuple should be a pair of strings.
+            First string is the attribute name you want to register
+            Second string is the description of the attribute.
+
+        Example
+        -------
+        >>> from sunpy.net import attr, attrs # doctest: +SKIP
+        >>> attr.Attr.update_values({attrs.Instrument: [('AIA', 'AIA is in Space.'),
+        ...                         ('HMI', 'HMI is next to AIA.')]}) # doctest: +SKIP
+        >>> attr.Attr._attr_registry[attrs.Instrument] # doctest: +SKIP
+        attr(name=['aia', 'hmi'], name_long=['AIA', 'HMI'],
+        ...  desc=['AIA is in Space.', 'HMI is next to AIA.']) # doctest: +SKIP
+        >>> attr.Attr._attr_registry[attrs.Instrument].name # doctest: +SKIP
+        ['aia', 'hmi']
+        >>> attr.Attr._attr_registry[attrs.Instrument].name_long # doctest: +SKIP
+        ['AIA', 'HMI']
+        >>> attr.Attr._attr_registry[attrs.Instrument].desc # doctest: +SKIP
+        ['AIA is in Space.', 'HMI is next to AIA.']
+        >>> attrs.Instrument.aia, attrs.Instrument.hmi # doctest: +SKIP
+        (<Instrument('AIA')>, <Instrument('HMI')>)
+        """
+
+        for key, value in adict.items():
+            if isiterable(value) and not isinstance(value, str):
+                for pair in value:
+                    if len(pair) != 2:
+                        raise ValueError(f'Invalid length (!=2) for values: {value}.')
+                    else:
+                        # Sanitize name, we remove all special characters and make it all lower case
+                        name = ''.join(char for char in pair[0] if char.isalnum()).lower()
+                        if keyword.iskeyword(name):
+                            # Attribute name has been appended with `_`
+                            # to make it a valid identifier since its a python keyword.
+                            name = name + '_'
+                        if not name.isidentifier():
+                            # This should account for names with one number first.
+                            # We match for single digits at the start only.
+                            if _REGEX.match(name):
+                                # This turns that digit into its name
+                                number = strtonum(name[0])
+                                name = number + "_" + name[1:]
```

Comment:
```suggestion
                                name = number + ("_" + name[1:] if len(name) > 1 else "")
```

We should only append the `_` if the name is not just one number i.e `0` for level 0